programmers/2일차/수박수박수박수.js

```diff
@@ -0,0 +1,23 @@
+function solution (n) {
+    const word  = "수박";
+    const split_word = word.split("");
+    const answer =[];
+    let count = 0;
+    while(n>count) {
+        for (let i of split_word) {
+            answer.push(i);
+            count += 1;
+            if ( count>= n) {
+                break;
+            }
+        } 
+    }
+    const join_answer = answer.join("");
+    return join_answer ;
```

Comment:
```suggestion
     return answer.join("");
   
```
변수를 선언해서 담을 필요없이 바로 return 해주시면 됩니다 
그리고 변수 선언을 하게 될 시에는 제일 상단에 미리 선언해주시는 것이 좋아요